dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/VoltageRegulators/VRRemote.mo

```diff
@@ -0,0 +1,36 @@
+within Dynawo.Electrical.Controls.Machines.VoltageRegulators;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model VRRemote "Model for centralized remote voltage regulation"
+
+import Dynawo.Connectors;
+import Dynawo.Types;
+
+  parameter Types.VoltageModulePu U0Pu "Start value of the regulated voltage in p.u (base UNom)";
```

Comment:
U0Pu should be protected and appear after parameters and public variables (coding style ;) )

Reply:
I'm note sure if I can put it as a protected parameter as it is directly used in the .par. What do you think ?

Reply:
Hm... In fact as you don't use an init model here I don't know...